running: {'--uuid': '17601751d452518dbe8c5431cf919d6c', '-db-root': '/home/ryedida/bbo_challenge_starter_kit/output', '--opt-root': '/home/ryedida/bbo_challenge_starter_kit/example_submissions', '--data-root': None, '--db': 'run_20230425_034200', '--opt': 'smoothness', '--data': 'diabetes', '--classifier': 'MLP-adam', '--metric': 'mse', '--calls': 15, '--suggestions': 1, '--jobs-file': None, '--verbose': False, 'dry_run': False, 'rev': 'a376313', 'opt_rev': 'x.x.x'}
cmd: python smoothness/optimizer.py -c MLP-adam -d diabetes -o smoothness -u 17601751d452518dbe8c5431cf919d6c -m mse -n 15 -p 1 -dir /home/ryedida/bbo_challenge_starter_kit/output -b run_20230425_034200
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

computed signature: ([{'alpha': 0.019628224813442792, 'batch_size': 182, 'beta_1': 0.9410202200271762, 'beta_2': 0.9998021557676793, 'epsilon': 1.8662266976518e-08, 'hidden_layer_sizes': 147, 'learning_rate_init': 0.0005627932047415167, 'tol': 0.03690557729213761, 'validation_fraction': 0.8846827852548593}, {'alpha': 0.0019982467392329444, 'batch_size': 200, 'beta_1': 0.919111482530466, 'beta_2': 0.9998488260436156, 'epsilon': 5.981221901152555e-07, 'hidden_layer_sizes': 61, 'learning_rate_init': 2.2310905607443014e-05, 'tol': 1.2046852412030316e-05, 'validation_fraction': 0.8117896445826539}, {'alpha': 0.46659545670218433, 'batch_size': 219, 'beta_1': 0.9889789783750891, 'beta_2': 0.9999896868093284, 'epsilon': 2.4234724484675948e-08, 'hidden_layer_sizes': 167, 'learning_rate_init': 2.972334644335654e-05, 'tol': 0.0036281404040243792, 'validation_fraction': 0.17260651658522078}, {'alpha': 4.656005689076002, 'batch_size': 135, 'beta_1': 0.870503881627747, 'beta_2': 0.9948873266941017, 'epsilon': 2.1023308743480125e-07, 'hidden_layer_sizes': 118, 'learning_rate_init': 0.0018781738757161913, 'tol': 1.1889379831773004e-05, 'validation_fraction': 0.6264327093752792}, {'alpha': 0.04705159350400542, 'batch_size': 158, 'beta_1': 0.9870884262730957, 'beta_2': 0.9999596874382349, 'epsilon': 1.1981845126013875e-08, 'hidden_layer_sizes': 116, 'learning_rate_init': 0.00617340520407431, 'tol': 1.741413418158619e-05, 'validation_fraction': 0.6754299026638921}], [29083.993548572587, 29062.423277324073, 29075.78987471409, 24626.32866834337, 9584.226858493177])
Signature errors:
                              0         1         2         3         4       max
MLP-adam_diabetes_mse  0.000007  0.000008  0.000003  0.003558  0.004952  0.004952
max                    0.000007  0.000008  0.000003  0.003558  0.004952  0.004952
starting sklearn study smoothness MLP-adam diabetes mse 15 1
with data root: None
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.493558 iter 0 next_points [{'alpha': 0.0016897827816802458, 'batch_size': 90, 'beta_1': 0.911195801848891, 'beta_2': 0.9478157527978788, 'epsilon': 6.748878555531614e-08, 'hidden_layer_sizes': 100, 'learning_rate_init': 0.06938904571138456, 'tol': 0.04940701516668456, 'validation_fraction': 0.19722773620838036}]
function_evaluation time 0.118139 value 3158.134853 suggestion {'alpha': 0.0016897827816802458, 'batch_size': 90, 'beta_1': 0.911195801848891, 'beta_2': 0.9478157527978788, 'epsilon': 6.748878555531614e-08, 'hidden_layer_sizes': 100, 'learning_rate_init': 0.06938904571138456, 'tol': 0.04940701516668456, 'validation_fraction': 0.19722773620838036}
observation time 0.000007, current best 3158.134853 at iter 0
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.329021 iter 1 next_points [{'alpha': 6.696644578771586e-05, 'batch_size': 211, 'beta_1': 0.8601693882433499, 'beta_2': 0.9870404356371761, 'epsilon': 2.2280575739343783e-09, 'hidden_layer_sizes': 150, 'learning_rate_init': 0.013373278121093868, 'tol': 2.8941723103794737e-05, 'validation_fraction': 0.1757097061785249}]
function_evaluation time 1.087566 value 2929.872420 suggestion {'alpha': 6.696644578771586e-05, 'batch_size': 211, 'beta_1': 0.8601693882433499, 'beta_2': 0.9870404356371761, 'epsilon': 2.2280575739343783e-09, 'hidden_layer_sizes': 150, 'learning_rate_init': 0.013373278121093868, 'tol': 2.8941723103794737e-05, 'validation_fraction': 0.1757097061785249}
observation time 0.000006, current best 2929.872420 at iter 1
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.332736 iter 2 next_points [{'alpha': 2.5306063063836303e-05, 'batch_size': 244, 'beta_1': 0.9508382113691858, 'beta_2': 0.9997188222655352, 'epsilon': 6.318316854198267e-07, 'hidden_layer_sizes': 56, 'learning_rate_init': 1.0675629772027256e-05, 'tol': 0.003952038077683673, 'validation_fraction': 0.8752932492238433}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.042058 value 29136.194264 suggestion {'alpha': 2.5306063063836303e-05, 'batch_size': 244, 'beta_1': 0.9508382113691858, 'beta_2': 0.9997188222655352, 'epsilon': 6.318316854198267e-07, 'hidden_layer_sizes': 56, 'learning_rate_init': 1.0675629772027256e-05, 'tol': 0.003952038077683673, 'validation_fraction': 0.8752932492238433}
observation time 0.000005, current best 2929.872420 at iter 2
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.344184 iter 3 next_points [{'alpha': 0.0013889609972435736, 'batch_size': 152, 'beta_1': 0.9890862102646429, 'beta_2': 0.9965802276892469, 'epsilon': 8.368769786889695e-09, 'hidden_layer_sizes': 164, 'learning_rate_init': 0.008996696847053103, 'tol': 4.79650534102303e-05, 'validation_fraction': 0.8362949065217389}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.669410 value 4243.603714 suggestion {'alpha': 0.0013889609972435736, 'batch_size': 152, 'beta_1': 0.9890862102646429, 'beta_2': 0.9965802276892469, 'epsilon': 8.368769786889695e-09, 'hidden_layer_sizes': 164, 'learning_rate_init': 0.008996696847053103, 'tol': 4.79650534102303e-05, 'validation_fraction': 0.8362949065217389}
observation time 0.000005, current best 2929.872420 at iter 3
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.324714 iter 4 next_points [{'alpha': 2.924594808077718e-05, 'batch_size': 94, 'beta_1': 0.5152223651327926, 'beta_2': 0.9999969205733665, 'epsilon': 2.7826996918303087e-08, 'hidden_layer_sizes': 92, 'learning_rate_init': 0.0011058157176988355, 'tol': 0.0009609655014086418, 'validation_fraction': 0.8889082634893446}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.048186 value 29079.412822 suggestion {'alpha': 2.924594808077718e-05, 'batch_size': 94, 'beta_1': 0.5152223651327926, 'beta_2': 0.9999969205733665, 'epsilon': 2.7826996918303087e-08, 'hidden_layer_sizes': 92, 'learning_rate_init': 0.0011058157176988355, 'tol': 0.0009609655014086418, 'validation_fraction': 0.8889082634893446}
observation time 0.000005, current best 2929.872420 at iter 4
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.346159 iter 5 next_points [{'alpha': 0.22334159093632455, 'batch_size': 150, 'beta_1': 0.9760666192895837, 'beta_2': 0.9969369739456065, 'epsilon': 1.2438113991974806e-09, 'hidden_layer_sizes': 64, 'learning_rate_init': 0.005398209682673895, 'tol': 0.040336272299660976, 'validation_fraction': 0.18335067118615395}]
function_evaluation time 0.061445 value 28772.214239 suggestion {'alpha': 0.22334159093632455, 'batch_size': 150, 'beta_1': 0.9760666192895837, 'beta_2': 0.9969369739456065, 'epsilon': 1.2438113991974806e-09, 'hidden_layer_sizes': 64, 'learning_rate_init': 0.005398209682673895, 'tol': 0.040336272299660976, 'validation_fraction': 0.18335067118615395}
observation time 0.000005, current best 2929.872420 at iter 5
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.328471 iter 6 next_points [{'alpha': 2.6724783887567107, 'batch_size': 35, 'beta_1': 0.9730729893294514, 'beta_2': 0.9995573556546071, 'epsilon': 2.7802349097520513e-07, 'hidden_layer_sizes': 92, 'learning_rate_init': 7.47807324291442e-05, 'tol': 0.0001747141057049252, 'validation_fraction': 0.518071843892536}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 1.086112 value 28881.782087 suggestion {'alpha': 2.6724783887567107, 'batch_size': 35, 'beta_1': 0.9730729893294514, 'beta_2': 0.9995573556546071, 'epsilon': 2.7802349097520513e-07, 'hidden_layer_sizes': 92, 'learning_rate_init': 7.47807324291442e-05, 'tol': 0.0001747141057049252, 'validation_fraction': 0.518071843892536}
observation time 0.000004, current best 2929.872420 at iter 6
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.459260 iter 7 next_points [{'alpha': 1.1331407853372808, 'batch_size': 191, 'beta_1': 0.5404989582482559, 'beta_2': 0.9736191603103649, 'epsilon': 2.6468834246198986e-09, 'hidden_layer_sizes': 98, 'learning_rate_init': 0.0016378729612259715, 'tol': 0.0003967578241600832, 'validation_fraction': 0.7605418450612094}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 0.668632 value 26392.205334 suggestion {'alpha': 1.1331407853372808, 'batch_size': 191, 'beta_1': 0.5404989582482559, 'beta_2': 0.9736191603103649, 'epsilon': 2.6468834246198986e-09, 'hidden_layer_sizes': 98, 'learning_rate_init': 0.0016378729612259715, 'tol': 0.0003967578241600832, 'validation_fraction': 0.7605418450612094}
observation time 0.000005, current best 2929.872420 at iter 7
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.324944 iter 8 next_points [{'alpha': 0.027317935100083738, 'batch_size': 189, 'beta_1': 0.9800009517023589, 'beta_2': 0.999959116486125, 'epsilon': 7.281883907021069e-07, 'hidden_layer_sizes': 60, 'learning_rate_init': 0.0011911309702463163, 'tol': 0.0008636068681013192, 'validation_fraction': 0.8983462087701489}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.049150 value 28953.517934 suggestion {'alpha': 0.027317935100083738, 'batch_size': 189, 'beta_1': 0.9800009517023589, 'beta_2': 0.999959116486125, 'epsilon': 7.281883907021069e-07, 'hidden_layer_sizes': 60, 'learning_rate_init': 0.0011911309702463163, 'tol': 0.0008636068681013192, 'validation_fraction': 0.8983462087701489}
observation time 0.000005, current best 2929.872420 at iter 8
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.322781 iter 9 next_points [{'alpha': 0.0009267535653042837, 'batch_size': 109, 'beta_1': 0.9844532079796466, 'beta_2': 0.9999986750443182, 'epsilon': 7.603578688528998e-08, 'hidden_layer_sizes': 50, 'learning_rate_init': 0.0009142454678520523, 'tol': 3.5451518005797914e-05, 'validation_fraction': 0.3694186753441081}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 0.571855 value 27317.302456 suggestion {'alpha': 0.0009267535653042837, 'batch_size': 109, 'beta_1': 0.9844532079796466, 'beta_2': 0.9999986750443182, 'epsilon': 7.603578688528998e-08, 'hidden_layer_sizes': 50, 'learning_rate_init': 0.0009142454678520523, 'tol': 3.5451518005797914e-05, 'validation_fraction': 0.3694186753441081}
observation time 0.000005, current best 2929.872420 at iter 9
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.326953 iter 10 next_points [{'alpha': 0.06271375272247046, 'batch_size': 137, 'beta_1': 0.6352188129629716, 'beta_2': 0.9502407269923372, 'epsilon': 1.6979769282634753e-09, 'hidden_layer_sizes': 76, 'learning_rate_init': 2.237965436456082e-05, 'tol': 0.0016728851432970353, 'validation_fraction': 0.624435671316911}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.052909 value 29123.656534 suggestion {'alpha': 0.06271375272247046, 'batch_size': 137, 'beta_1': 0.6352188129629716, 'beta_2': 0.9502407269923372, 'epsilon': 1.6979769282634753e-09, 'hidden_layer_sizes': 76, 'learning_rate_init': 2.237965436456082e-05, 'tol': 0.0016728851432970353, 'validation_fraction': 0.624435671316911}
observation time 0.000005, current best 2929.872420 at iter 10
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.327394 iter 11 next_points [{'alpha': 0.024505698977470243, 'batch_size': 188, 'beta_1': 0.763813701845291, 'beta_2': 0.9152099955072115, 'epsilon': 5.530205406876957e-09, 'hidden_layer_sizes': 58, 'learning_rate_init': 1.2358736432269017e-05, 'tol': 0.000814018527101972, 'validation_fraction': 0.1833821962464456}]
function_evaluation time 0.061333 value 29132.624084 suggestion {'alpha': 0.024505698977470243, 'batch_size': 188, 'beta_1': 0.763813701845291, 'beta_2': 0.9152099955072115, 'epsilon': 5.530205406876957e-09, 'hidden_layer_sizes': 58, 'learning_rate_init': 1.2358736432269017e-05, 'tol': 0.000814018527101972, 'validation_fraction': 0.1833821962464456}
observation time 0.000005, current best 2929.872420 at iter 11
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.326007 iter 12 next_points [{'alpha': 0.4407498970611122, 'batch_size': 17, 'beta_1': 0.6251197351730695, 'beta_2': 0.9999975750014487, 'epsilon': 9.535755287615439e-08, 'hidden_layer_sizes': 178, 'learning_rate_init': 0.000944474202381062, 'tol': 0.011900063103954948, 'validation_fraction': 0.8445365225008579}]
function_evaluation time 0.084702 value 28889.246307 suggestion {'alpha': 0.4407498970611122, 'batch_size': 17, 'beta_1': 0.6251197351730695, 'beta_2': 0.9999975750014487, 'epsilon': 9.535755287615439e-08, 'hidden_layer_sizes': 178, 'learning_rate_init': 0.000944474202381062, 'tol': 0.011900063103954948, 'validation_fraction': 0.8445365225008579}
observation time 0.000005, current best 2929.872420 at iter 12
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.325697 iter 13 next_points [{'alpha': 0.252460035629128, 'batch_size': 223, 'beta_1': 0.7923502776051237, 'beta_2': 0.9997941118082706, 'epsilon': 1.1821203139916101e-07, 'hidden_layer_sizes': 173, 'learning_rate_init': 0.0002737979407120933, 'tol': 0.00011204149681100464, 'validation_fraction': 0.8295685810674902}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 0.765896 value 28748.591290 suggestion {'alpha': 0.252460035629128, 'batch_size': 223, 'beta_1': 0.7923502776051237, 'beta_2': 0.9997941118082706, 'epsilon': 1.1821203139916101e-07, 'hidden_layer_sizes': 173, 'learning_rate_init': 0.0002737979407120933, 'tol': 0.00011204149681100464, 'validation_fraction': 0.8295685810674902}
observation time 0.000004, current best 2929.872420 at iter 13
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.323125 iter 14 next_points [{'alpha': 2.122096831290245e-05, 'batch_size': 214, 'beta_1': 0.9794649773207657, 'beta_2': 0.976814677474638, 'epsilon': 2.0275356445492593e-07, 'hidden_layer_sizes': 126, 'learning_rate_init': 0.016169147208037627, 'tol': 0.06935283574399548, 'validation_fraction': 0.4140056640552755}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.060784 value 27923.423894 suggestion {'alpha': 2.122096831290245e-05, 'batch_size': 214, 'beta_1': 0.9794649773207657, 'beta_2': 0.976814677474638, 'epsilon': 2.0275356445492593e-07, 'hidden_layer_sizes': 126, 'learning_rate_init': 0.016169147208037627, 'tol': 0.06935283574399548, 'validation_fraction': 0.4140056640552755}
observation time 0.000005, current best 2929.872420 at iter 14
saving meta data: {'args': {'--uuid': '17601751d452518dbe8c5431cf919d6c', '-db-root': '/home/ryedida/bbo_challenge_starter_kit/output', '--opt-root': '/home/ryedida/bbo_challenge_starter_kit/example_submissions', '--data-root': None, '--db': 'run_20230425_034200', '--opt': 'smoothness', '--data': 'diabetes', '--classifier': 'MLP-adam', '--metric': 'mse', '--calls': 15, '--suggestions': 1, '--jobs-file': None, '--verbose': False, 'dry_run': False, 'rev': 'a376313', 'opt_rev': 'x.x.x'}, 'signature': ([{'alpha': 0.019628224813442792, 'batch_size': 182, 'beta_1': 0.9410202200271762, 'beta_2': 0.9998021557676793, 'epsilon': 1.8662266976518e-08, 'hidden_layer_sizes': 147, 'learning_rate_init': 0.0005627932047415167, 'tol': 0.03690557729213761, 'validation_fraction': 0.8846827852548593}, {'alpha': 0.0019982467392329444, 'batch_size': 200, 'beta_1': 0.919111482530466, 'beta_2': 0.9998488260436156, 'epsilon': 5.981221901152555e-07, 'hidden_layer_sizes': 61, 'learning_rate_init': 2.2310905607443014e-05, 'tol': 1.2046852412030316e-05, 'validation_fraction': 0.8117896445826539}, {'alpha': 0.46659545670218433, 'batch_size': 219, 'beta_1': 0.9889789783750891, 'beta_2': 0.9999896868093284, 'epsilon': 2.4234724484675948e-08, 'hidden_layer_sizes': 167, 'learning_rate_init': 2.972334644335654e-05, 'tol': 0.0036281404040243792, 'validation_fraction': 0.17260651658522078}, {'alpha': 4.656005689076002, 'batch_size': 135, 'beta_1': 0.870503881627747, 'beta_2': 0.9948873266941017, 'epsilon': 2.1023308743480125e-07, 'hidden_layer_sizes': 118, 'learning_rate_init': 0.0018781738757161913, 'tol': 1.1889379831773004e-05, 'validation_fraction': 0.6264327093752792}, {'alpha': 0.04705159350400542, 'batch_size': 158, 'beta_1': 0.9870884262730957, 'beta_2': 0.9999596874382349, 'epsilon': 1.1981845126013875e-08, 'hidden_layer_sizes': 116, 'learning_rate_init': 0.00617340520407431, 'tol': 1.741413418158619e-05, 'validation_fraction': 0.6754299026638921}], [29083.993548572587, 29062.423277324073, 29075.78987471409, 24626.32866834337, 9584.226858493177])}
saving results
saving timing
saving suggest log
done
